symengine/cwrapper.cpp

```diff
@@ -410,6 +438,20 @@ int is_a_ComplexDouble(const basic c)
 {
     return is_a<ComplexDouble>(*(c->m));
 }
+int is_a_RealMPFR(const basic c)
+{
+#ifdef HAVE_SYMENGINE_MPFR
+    return is_a<RealMPFR>(*(c->m));
+#endif // HAVE_SYMENGINE_MPFR
+    return false;
+}
+int is_a_ComplexMPC(const basic c)
+{
+#ifdef HAVE_SYMENGINE_MPC
+    return is_a<ComplexMPC>(*(c->m));
+#endif // HAVE_SYMENGINE_MPC
+    return false;
```

Comment:
Use an `else` here, otherwise the compiler spits out warnings about unreachable code